pubnub-kotlin/pubnub-kotlin-api/src/jvmMain/kotlin/com/pubnub/api/PubNub.kt

```diff
@@ -774,6 +774,8 @@ actual interface PubNub : StatusEmitter, EventEmitter {
         channelGroups: List<String>,
         includeState: Boolean,
         includeUUIDs: Boolean,
+        limit: Int,
```

Comment:
Same question about optional value (if internally we enforce `1000`).

Reply:
In kotlin/pubnub-kotlin/pubnub-kotlin-api/src/commonMain/kotlin/com/pubnub/api/PubNub.kt there is proper definition:

fun hereNow(
    channels: List<String> = emptyList(),
    channelGroups: List<String> = emptyList(),
    includeState: Boolean = false,
    includeUUIDs: Boolean = true,
    limit: Int = 1000,
    offset: Int? = null,
): HereNow